### PR TITLE
fix: Add a default value for dependabot_filename_to_use

### PR DESCRIPTION
**Pull request by gbrindisi**
# Pull Request
Fix https://github.com/github/evergreen/issues/168

## Proposed Changes
Set a default value to dependabot_filename_to_use rather than `None`

### Author/Contributor

- [x] If documentation is needed for this change, has that been included in this pull request
- [x] run `make lint` and fix any issues that you have introduced
- [x] run `make test` and ensure you have test coverage for the lines you are introducing
- [ ] If publishing new data to the public (scorecards, security scan results, code quality results, live dashboards, etc.), please request review from `@jeffrey-luszcz`

### Reviewer

- [x] Label as either `fix`, `documentation`, `enhancement`, `infrastructure`, `maintenance` or `breaking`

### CODE REVIEW

File: evergreen.py [hunk 1]
```diff
@@ -80,8 +80,8 @@ def main():  # pragma: no cover
             print("Skipping " + repo.full_name + " (visibility-filtered)")
             continue
         existing_config = None
-        filename_list = [".github/dependabot.yml", ".github/dependabot.yaml"]
-        dependabot_filename_to_use = None
+        filename_list = [".github/dependabot.yaml", ".github/dependabot.yml"]
```
Comment (jmeridth) [inline evergreen.py:83]: I'm good with this idea but why flip the array order?

Comment (gbrindisi) [reply evergreen.py:83]: It is the recommended extension AFAIK, with `yml` being sort of legacy.

https://yaml.org/faq.html

But it doesn't really matter if you rather not have the list flipped I can remove this part.

Comment (jmeridth) [reply evergreen.py:83]: > It is the recommended extension AFAIK, with `yml` being sort of legacy.
> 
> 
> 
> https://yaml.org/faq.html
> 
> 
> 
> But it doesn't really matter if you rather not have the list flipped I can remove this part.

Cool. Was just curious. Both are detected. Thanks.

**Review by zkoppert — APPROVED** [review]
Awesome! Thank you @gbrindisi !

**Pull request merged**